Automattic/wp-calypso
# Dialog: migrate CSS to webpack

Migrates `Dialog` styles to webpack and merges the `DialogBase` component into `Dialog`. The result is just one `Dialog` component instead of two.

**How to test:**
- verify that all modals still work and look good
- verify that the removed transition styles are really not used anywhere (most probably with `react-transition-group`)
